Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -855,6 +855,29 @@ class BrowserViewController: UIViewController {
         }
         tabManager.selectTab(tab)
     }
+
+    func presentActivityViewController(url: NSURL, tab: Browser? = nil, sourceView: UIView, sourceRect: CGRect, arrowDirection: UIPopoverArrowDirection) {
+        let helper = ShareExtensionHelper(url: url, tab: tab)
+
+        let controller = helper.createActivityViewController({
+            // We don't know what share action the user has chosen so we simply always
+            // update the toolbar and reader mode bar to refelect the latest status.
```

Comment:
Ha, sorry, I already had it corrected and then apparently copied the typo over again when resolving the conflicts during rebasing.